src/renderers/webgl/WebGLPrograms.js

```diff
@@ -149,6 +152,14 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 			iridescenceMap: useIridescence && !! material.iridescenceMap,
 			iridescenceThicknessMap: useIridescence && !! material.iridescenceThicknessMap,
 
+			sheen: useSheen,
+			sheenColorMap: useSheen && !! material.sheenColorMap,
+			sheenRoughnessMap: useSheen && !! material.sheenRoughnessMap,
+
+			transmission: useTransmission,
+			transmissionMap: useTransmission && !! material.transmissionMap,
+			thicknessMap: useTransmission && !! material.thicknessMap,
```

Comment:
Is `thicknessMap` used when `transmission` is not used?

Reply:
At least in glTF "thicknessMap" comes from the [volume extension](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_volume/README.md#dependencies) that requires the transmission extension